controllers/transactions.js

```diff
@@ -0,0 +1,33 @@
+const transactionsModel = require('../models/transaction')
+const { getUserId } = require('../utils/users')
+
+/**
+ * Collects all transactions and sends only required data for username spicified in url
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const fetchTransactionsByUserId = async (req, res) => {
+  try {
+    const userId = await getUserId(req.params.username)
+    if (userId) {
+      const orderBy = req.query.orderBy || 'DESC'
+      const offset = parseInt(req.query.offset, 10) || 0
+      const limit = parseInt(req.query.limit, 10) + offset || 10
+      const data = await transactionsModel.fetchTransactionsByUserId(userId, offset, limit, orderBy)
```

Comment:
Can we get the limit from a const file instead of hard coding it?

Reply:
Can you please it how we can do that?

Reply:
Can you please explain i don't know how to do that?

Reply:
Done

Reply:
If you have done the changes, please push the code, Code has not been updated